### PR TITLE
Styles, classnames

### PR DESCRIPTION
**Pull request by fabiosantoscode**
This styles the animatedpanel according to the given designs.

A demo of this is available at http://fabiosantoscode.github.io/component-animatedpanel

### CODE REVIEW

**Comment by fabiosantoscode** [comment]
Hint: if it seems like it doesn't work, you probably forgot you have an ad blocker ;)

**Comment by angelorepole** [comment]
Is this animated panel only usable for ads?

**Comment by angelorepole** [comment]
the problem I have with this component is the name. This is a ad display component
I know the name comes from WI but I believe the idea was to have a general component that can display items with an animation effect.

so either we generalise it so things like this  end up in props  title = (<span ref="title" className="animatedpanel__title">Advertisement</span>)

or we need to change the name

**Comment by angelorepole** [comment]
Can we please add https://schema.org/WPAdBlock also I guess we could mark this as role="complementary" for accessibility

**Comment by fabiosantoscode** [comment]
@angelorepole this has to be an exclusive ad display thing, there will be too many props only for ad display, setting zones, and responsiveness. And we do need to display the "advertisement" word.

I'm going to do WPAdBlock now.

**Comment by fabiosantoscode** [comment]
So yes, we must rename this thing. It was always ad-only.

**Comment by keithamus** [comment]
@fabiosantoscode yes, it was originally badly named. It is an ad-panel not an animated-panel. Feel free to rename in the package.json and when we merge we'll rename it

**Comment by angelorepole** [comment]
Thanks for the changes. Approved. @keithamus is this good for you. If yes I can (or you) merge.

**Comment by fabiosantoscode** [comment]
@keithamus rename done, so you guys can go ahead and merge/rename dis (I don't have permissions for this repo)

File: test/index.es6 [hunk 1]
```diff
@@ -97,7 +97,7 @@ describe('AnimatedPanel', () => {
 
   });
 
-  describe('showElementWhenInView', () => {
+  describe('loadElementWhenInView', () => {
 
 
   });
```
Comment (keithamus) [inline test/index.es6]: Is this an empty describe?

**Comment by keithamus** [comment]
Good work, we need more tests though.

**Pull request merged**